.github/dependabot.yml

```diff
@@ -0,0 +1,17 @@
+version: 2
+
+updates:
+
+  # Maintain dependencies for GitHub Actions
+- package-ecosystem: github-actions
+  directory: /
+  schedule:
+    interval: weekly
```

Comment:
Figured we might as well add these, too.

Reply:
Yay for automation!